lib/compat/wordpress-5.9/get-global-styles-and-settings.php

```diff
@@ -156,17 +156,10 @@ function wp_get_global_styles_svg_filters() {
 	}
 
 	$supports_theme_json = WP_Theme_JSON_Resolver_Gutenberg::theme_has_support();
-	$supports_link_color = get_theme_support( 'experimental-link-color' );
-
-	// TODO: Which origins are needed for duotone filters?
-	$origins = array( 'core', 'theme', 'user' );
-	if ( ! $supports_theme_json && ! $supports_link_color ) {
-		// In this case we only enqueue the core presets (CSS Custom Properties + the classes).
-		$origins = array( 'core' );
-	} elseif ( ! $supports_theme_json && $supports_link_color ) {
-		// For the legacy link color feature to work, the CSS Custom Properties
-		// should be in scope (either the core or the theme ones).
-		$origins = array( 'core', 'theme' );
+
+	$origins = array( 'default', 'theme' );
```

Comment:
The `core` key has changed to `default`, that's why the default presets were not working.